### PR TITLE
Removed some repetition.

### PR DESCRIPTION
**Pull request by krstffr**
Only tested in chrome dev tools though, so be careful :)

### CODE REVIEW

File: src/helpers/arrow-styles.js [hunk 1]
```diff
@@ -1,45 +1,35 @@
...
-		width: "0px",
-		height: "0px",
-		borderLeft: `${size}px solid transparent`,
-		borderRight: `${size}px solid transparent`,
-		borderBottom: `${size}px solid ${color}`,
-		position: "absolute",
-		top: `-${size}px`
-	};
+const defaultArrow = {
+  width: '0px',
```
Comment (conorhastings) [inline src/helpers/arrow-styles.js]: can you use double quotes here

File: src/helpers/arrow-styles.js [hunk 2]
```diff
@@ -1,45 +1,35 @@
...
-		height: "0px",
-		borderTop: `${size}px solid transparent`,
-		borderBottom: `${size}px solid transparent`,
-		borderRight: `${size}px solid ${color}`,
-		position: "absolute",
-		left: `-${size}px`
-	};
-};
+export const arrowRight = ({ color, size }) =>
+Object.assign({}, defaultArrow, {
```
Comment (conorhastings) [inline src/helpers/arrow-styles.js]: can you wrap this in a `()`

File: src/helpers/arrow-styles.js [hunk 3]
```diff
@@ -1,45 +1,35 @@
...
-	return {
-		width: "0px",
-		height: "0px",
-		borderTop: `${size}px solid transparent`,
-		borderBottom: `${size}px solid transparent`,
-		borderLeft: `${size}px solid ${color}`,
-		position: "absolute"
-	};
-};
+export const arrowDown = ({ color, size }) =>
```
Comment (conorhastings) [inline src/helpers/arrow-styles.js]: can you wrap this in a `()`

File: src/helpers/arrow-styles.js [hunk 4]
```diff
@@ -1,45 +1,35 @@
...
-	return {
-		width: "0px",
-		height: "0px",
-		borderLeft: `${size}px solid transparent`,
-		borderRight: `${size}px solid transparent`,
-		borderTop: `${size}px solid ${color}`,
-		position: "absolute"
-	};
-};
+export const arrowUp = ({ color, size }) =>
```
Comment (conorhastings) [inline src/helpers/arrow-styles.js]: can you wrap this in a `()`

File: src/helpers/arrow-styles.js [hunk 5]
```diff
@@ -1,45 +1,35 @@
...
-	};
-};
+export const arrowRight = ({ color, size }) =>
+Object.assign({}, defaultArrow, {
+  borderTop: `${size}px solid transparent`,
+  borderBottom: `${size}px solid transparent`,
+  borderLeft: `${size}px solid ${color}`,
+});
+
+export const arrowLeft = ({ color, size }) =>
```
Comment (conorhastings) [inline src/helpers/arrow-styles.js]: can you wrap this in a `()`

File: src/helpers/arrow-styles.js (hunk 4 above)
Comment (krstffr) [inline src/helpers/arrow-styles.js]: wrap what exactly?

File: src/helpers/arrow-styles.js (hunk 1 above)
Comment (krstffr) [inline src/helpers/arrow-styles.js]: Sure, the airbnb styleguide suggests single quotes though, but maybe you use some other style guide?

https://github.com/airbnb/javascript#strings--quotes

Comment (conorhastings) [inline src/helpers/arrow-styles.js]: yes, we use an internal style guide, probably would be helpful to just add an eslint here

File: src/helpers/arrow-styles.js (hunk 4 above)
Comment (conorhastings) [inline src/helpers/arrow-styles.js]:
```export const arrowRight = ({ color, size }) => (
	Object.assign({}, defaultArrow, {
	  borderTop: `${size}px solid transparent`,
	  borderBottom: `${size}px solid transparent`,
	  borderLeft: `${size}px solid ${color}`,
	})
);
```

**Comment by krstffr** [comment]
Does that do it?

**Comment by conorhastings** [comment]
looks good thanks!

**Pull request merged**